addons/im_livechat/static/tests/chat_window_patch.test.js

```diff
@@ -44,3 +47,41 @@ test("closing a chat window with no message from admin side unpins it", async ()
     await contains(".o-mail-DiscussSidebarChannel", { text: "Partner 1" });
     await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "Partner 2" });
 });
+
+test("Focus should not be stolen when a new livechat open", async () => {
+    const pyEnv = await startServer();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor 12" });
+    const channelIds = pyEnv["discuss.channel"].create([
+        { name: "general" },
+        {
+            anonymous_name: "Visitor 12",
+            channel_member_ids: [
+                Command.create({
+                    partner_id: serverState.partnerId,
+                }),
+                Command.create({ guest_id: guestId }),
+            ],
+            channel_type: "livechat",
+            livechat_active: true,
+        },
+    ]);
+    await start();
+    await click(".o_menu_systray i[aria-label='Messages']");
+    await click(".o-mail-NotificationItem", { text: "general" });
+    await contains(".o-mail-ChatWindow", { text: "general" });
+    await contains(".o-mail-Composer-input[placeholder='Message #general…']:focus");
+    withGuest(guestId, () =>
+        rpc("/mail/message/post", {
+            post_data: {
+                body: "hu",
+                message_type: "comment",
+                subtype_xmlid: "mail.mt_comment",
+            },
+            thread_id: channelIds[1],
+            thread_model: "discuss.channel",
+        })
+    );
+    await contains(".o-mail-ChatWindow", { text: "Visitor 12" });
+    await animationFrame();
+    await contains(".o-mail-ChatWindow:contains('general') .o-mail-Composer-input:focus");
```

Comment:
This should be the same assert as above

`await contains(".o-mail-Composer-input[placeholder='Message #general…']:focus");`